src/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:centos8.3.2011
+
+LABEL org.opencontainers.image.title="huawei-csi-plugin"
+LABEL org.opencontainers.image.description="A docker image containing the Huawei CSI plugin for k8s"
+LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
+LABEL org.opencontainers.image.vendor="Adfinis"
+LABEL org.opencontainers.image.url="https://github.com/adfinis-sygroup/huawei-csi-plugin"
+LABEL org.opencontainers.image.licenses="AGPL-3.0-or-later"
+
+RUN yum install -y \ 
+device-mapper-multipath \ 
+iscsi-initiator-utils \ 
+nvme-cli \
+xfsprogs \ 
+nfs-utils \ 
+net-tools \ 
+e2fsprogs 
+
+ADD ["bin/huawei-csi", "/"] 
+ADD ["bin/passwdEncrypt", "/"] 
```

Comment:
Comitting binaries is rather unusual, did you consider downloading the binaries as part of the build process.

Also, the fact that the binaries are not signed and we don't have any checksums for them isn't very reassuring :(

Reply:
It is. We would rather compile the binaries in the build process in a second step. For now this is how it is.